app/assets/stylesheets/desktop/topic-post.scss

```diff
@@ -493,6 +493,20 @@ blockquote {
   padding: 12px;
 }
 
+aside.quote:has(.title) .title {
+  border-top-right-radius: var(--d-border-radius);
+  border-top-left-radius: var(--d-border-radius);
+}
+
+aside.quote:has(.title) blockquote {
+  border-bottom-right-radius: var(--d-border-radius);
+  border-bottom-left-radius: var(--d-border-radius);
+}
+
+aside.quote:not(:has(.title)) blockquote {
```

Comment:
I believe it's possible to avoid `:has` here (I just did something along those lines a few days ago in my theme)

```scss
aside.quote blockquote {
  border-radius: var(--d-border-radius);
}

aside.quote .title + blockquote {
  border-top-left-radius: 0;
  border-top-right-radius: 0;
}
```

Reply:
Also, a question: should a standalone `blockquote` (or `.cooked blockquote` to be slightly more specific) also have border-radius? Those are used when you type `> `, I see them daily on dev.

Reply:
Didnt see that I had any comments here so sorry for the delay. Thanks for all of the suggestions, Ill implement then ask for a review.